generators/app/templates/scripts/start.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+show_help() {
+  echo "Usage: ./start.sh"
+  echo ""
+  echo "-d or --down delete all container"
+  echo "-wp or --windows-path convert to Windows path"
+  echo "-wt or --wait-time wait for backend in s. <= 0 no wait at all. Default 500s"
+  echo "-h or --help"
+}
+
+set_windows_path(){
+  export COMPOSE_CONVERT_WINDOWS_PATHS=1
```

Comment:
Where is used this ENV Variable in Docker Compose / Generator?

Reply:
I added the link to the explanation of that env in the PR description